pkg/component/seedsystem/seedsystem.go

```diff
@@ -103,13 +106,14 @@ func (s *seedSystem) WaitCleanup(ctx context.Context) error {
 }
 
 func (s *seedSystem) computeResourcesData() (map[string][]byte, error) {
-	var (
-		registry = managedresources.NewRegistry(kubernetes.SeedScheme, kubernetes.SeedCodec, kubernetes.SeedSerializer)
-	)
+	var registry = managedresources.NewRegistry(kubernetes.SeedScheme, kubernetes.SeedCodec, kubernetes.SeedSerializer)
 
 	if s.values.ReserveExcessCapacity.Enabled {
-		if err := s.addReserveExcessCapacityDeployment(registry); err != nil {
-			return nil, err
+		for i, config := range s.values.ReserveExcessCapacity.Configs {
+			name := fmt.Sprintf("reserve-excess-capacity-%d", i)
+			if err := s.addReserveExcessCapacityDeployment(registry, name, config); err != nil {
+				return nil, err
+			}
```

Comment:
Shouldn't we add the current/standard deployment by default? Now, it's a breaking change (not API-wise, but behaviour-wise) because people need to specify the new `configs` field if they still want the standard deployment. This also has some benefits (people can "fine-tune" it and/or change the resources, for example), of course. I don't read anything in the description/release note about it.

Reply:
[Defaulting will set a default config](https://github.com/gardener/gardener/blob/2e9f80e9e493be9d1c95ae332cc6de8a6efb27a2/pkg/apis/core/v1beta1/defaults.go#L65-L67) with the CPU and memory reservations of the current deployment when `enabled=true` and `configs` are empty, so its behavior should not change.

Reply:
OK 👍🏻